tools/interactive/interactivetool_odv.xml

```diff
@@ -0,0 +1,38 @@
+<tool id="interactive_tool_odv" tool_type="interactive" name="ODV" version="@VERSION@">
+    <description>interactive plotting tool for geo-referenced data</description>
+    <macros>
+    <token name="@VERSION@">5.6.5</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/mariejo/docker_odv</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="ODV on $infile.display_name" requires_domain="True">
+            <port>80</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <!-- see: https://odv.awi.de/fileadmin/user_upload/odv/misc/odvGuide.pdf -->
+    <![CDATA[
+    	mkdir output &&
+        mkdir -p /root/Documents/ODV/data/output &&
+        cp '$infile' '/root/Documents/ODV/data/$infile.display_name' &&
+        bash /startup.sh
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="netcdf,h5" label="netcdf"/>
+    </inputs>
+    <outputs>
+ 	<data name="version" format="txt" label="${tool.name} on ${on_string}: version.txt" from_work_dir="version.txt" />
+        <collection name="outputs" type="list" label="ODV outputs">
+            <discover_datasets pattern="__name_and_ext__" directory="output" />
+        </collection>
+    </outputs>
+    <tests>
+    </tests>
+    <help><![CDATA[
+        `ODV <https://odv.awi.de/>`_ plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets.
+    ]]>
+    </help>
```

Comment:
Do you want to add maybe a citation for the tool?

Reply:
Ah maybe yes I will search for the right one. Thanks !!